toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md

```diff
@@ -275,55 +278,58 @@ Finally, please note that this is a larger set of apps than were required for 1.
-| Test  | 1. Raise        | `fdc3.raiseIntent(‘aTestingIntent1’, testContextX)`<br>starts app A.                       |
-| A     | 2. Receive Intent & Context | After starting up, A runs `fdc3.addIntentListener(‘aTestingIntent1’)` to register its listener.<br>It then receives `testContextY`, matching that sent by Test |
+| Test  | 1. Raise        | `fdc3.raiseIntent("aTestingIntent", testContextX)`<br>starts app A.                       |
+| A     | 2. Receive Intent & Context | After starting up, A runs `fdc3.addIntentListener("aTestingIntent1")` to register its listener.<br>It then receives `testContextX`, matching that sent by Test |
 | Test  | 3. IntentResolution          | The `raiseIntent` call returns an `IntentResolution` Object with an `AppIdentifer` as the `source field` with App A's `appId` and `instanceId` set.**                       |
 
 - `2.0-RaiseIntentSingleResolve`: Perform above test
-- `2.0-RaiseIntentTargetedAppResolve`: Repeat the above test, but: 
-  - In the first step use `fdc3.raiseIntent(‘aTestingIntent’, testContextY, {"appID": "<Y’s appId>"})` to start app A, 
+- `2.0-RaiseIntentTargetedAppResolve`: Repeat the above test, but:
+  - In the first step use `fdc3.raiseIntent("sharedTestingIntent1", testContextX, {"appID": "<B’s appId>"})` to start app B,
   - Otherwise, as above.
-- `2.0-RaiseIntentTargetedInstanceResolveOpen`: Repeat the above test, but:** 
+- `2.0-RaiseIntentTargetedInstanceResolveOpen`: Repeat the above test, but:
   - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
-  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+  - Then in the first step, use `fdc3.raiseIntent("aTestingIntent", testContextX, appIdentifier)` to target the running instance of app A. 
+  - Confirm that the intent is delivered to the correct instance and that another instance is NOT started. Otherwise, as above.
 - `2.0-RaiseIntentTargetedInstanceResolveFindInstances`: Repeat the above test, but:
-  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
-  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A.
+  - Then use `const instances = await fdc3.findInstances({appId: "<A's appId>"})` to retrieve a list of instances of app A. Confirm that only one is present and retrieve its `AppIdentifier`, confirming that it contains and `instanceId` field that matches that returned by the `fdc3.open` call.
```

Comment:
minor typo. 'and' should be 'an' -> '...confirming that it contains and `instanceId`...'

Reply:
:+1 fixed